src/components/StoryPage/index.js

```diff
@@ -30,7 +30,7 @@ function StoryPage({
         <Grid container>
           <Grid item xs={false} lg={1} />
```

Comment:
```suggestion
          <Grid item xs={undefined} lg={1} />
```
`xs`, `sm`, etc. can't be false/true.